### ISSUE TITLE
Attribute `aria-label` is not translated

### ISSUE THREAD

**Issue by luisg123v** [odoo/odoo#29699]
Impacted versions:
- 12.0

Steps to reproduce:
- Change the language to any different from English, e.g. Spanish
- Inspect the Activities menu

Current behavior:
The `title` attribute is translated, but `aria-label` is not:

```
<a ... href="#" role="button" title="Actividades">
    <i aria-label="Activities" class="fa fa-clock-o" role="img"/>
...
</a>
```

Expected behavior:
Both `title` and `aria-label` should be translated.

**Comment by mart-e** [comment]
@Julien00859 haven't you fixed that one?

**Comment by Julien00859** [comment]
Not yet but there is a fix going on https://github.com/odoo/odoo/pull/29432

**Comment by Julien00859** [comment]
The issue is known and efforts are going on to solve it already, see linked pull request. Thanks anyway for reporting this issue :+1:

**Comment by luisg123v** [comment]
Fixed on https://github.com/odoo/odoo/pull/48255
